libs/design-system/src/typography/text/Text.tsx

```diff
@@ -13,6 +13,7 @@ interface ITextProps extends JSX.ElementChildrenAttribute, SpacingProps {
   color?: MantineColor;
   rows?: number;
   gradient?: boolean;
+  transform?: TextProps['transform'];
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Great work on using strong types here!‏